### PR TITLE
Fix#1186 Fix Va-data-table exporting, add overflow-x behavior

### PR DESCRIPTION
**Pull request by RVitaly1978**
## Description

close: #1186

Updated:
- [x] - add VaDataTable to vuestic-components
- [x] - add overflow-x behavior to table

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-data-table/VaDataTable.vue [hunk 1]
```diff
@@ -428,133 +432,140 @@ export default defineComponent({
 @import "./variables";
 // The calculated variables are taken from a respective element's `style` attribute. See the `useStyleable` hook
 
-.va-data-table {
-  width: 100%;
-  cursor: default;
+.va-data-table__wrapper {
```
Comment (asvae) [inline packages/ui/src/components/va-data-table/VaDataTable.vue:435]: That's pretty weird nesting bem-wise

**Comment by asvae** [comment]
Lef small comment, apart from that - good for merge!

File: packages/ui/src/components/va-data-table/VaDataTable.vue (hunk 1 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-data-table/VaDataTable.vue:435]: Fixed: removed the nesting of css-selectors.

**Comment by asvae** [comment]
Thanks!

**Pull request merged**